daemon/src/collab_settlement/maker.rs

```diff
@@ -156,7 +155,9 @@ impl Actor {
 
                     let DialerSignature { dialer_signature } = framed
                         .next()
-                        .timeout(SETTLEMENT_MSG_TIMEOUT, || tracing::debug_span!("receive dialer signature"))
+                        .timeout(SETTLEMENT_MSG_TIMEOUT, || {
```

Comment:
Are these changes from dprint or manual?

Reply:
It was `dprint`. 

Funnily enough it only happens when I save the files using emacs, which calls `dprint` via [`dprint-fmt`](https://github.com/luckysori/dprint-fmt).

Sorry about the diff noise!

Reply:
Huh, that's a bit weird :thinking: I wonder how it originally passed CI without being properly formatted...

Reply:
My guess is that the workspace-level call to `dprint check` is okay with the file either way.

And the [file-level call](https://github.com/luckysori/dprint-fmt/blob/88ab37467a4a7a7d7660297b462b676239508485/dprint-fmt.el#L50) to `dprint fmt -stdin <file-path>` using the `dprint-fmt` emacs package sees things differently here, which is very weird but mostly harmless.